mail-client/alot/alot-0.9.1.ebuild

```diff
@@ -37,22 +37,19 @@ DEPEND="
 	"
 
 PATCHES=(
-	"${FILESDIR}/${PV}-0001-remove-non-working-test.patch"
-	"${FILESDIR}/${PV}-0002-changed-expired-test-keys.patch"
+	"${FILESDIR}/0.9-0001-remove-non-working-test.patch"
 	)
 
-python_compile_all() {
-	if use doc; then
-		emake -C docs html
-		HTML_DOCS=( docs/build/html/. )
-	fi
-}
+distutils_enable_tests unittest
 
-src_test() {
-	esetup.py test
+python_compile_all() {
+	emake -C docs man
+	use doc && emake -C docs html
 }
 
 python_install_all() {
+	use doc && local HTML_DOCS=( docs/build/html/. )
+	doman docs/build/man/*
 	distutils-r1_python_install_all
 
 	insinto /usr/share/alot
```

Comment:
These lines can be replaced by a call to distutils_enable_sphinx

Reply:
Unfortunately no.